runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java

```diff
@@ -0,0 +1,35 @@
+package net.runelite.client.plugins.xptracker;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	keyName = "xpTracker",
+	name = "XP Tracker",
+	description = "Configuration for the XP Tracker plugin"
+)
+public interface XpTrackerConfig extends Config
+{
+	@ConfigItem(
+		position = 0,
+		keyName = "logoutPausing",
+		name = "Logout Pausing",
+		description = "Configures what time should count towards XP rates, +#m means # minutes of log out time also counts"
+	)
+	default XpAutoPauseLogoutBehavior logoutPausing()
+	{
+		return XpAutoPauseLogoutBehavior.NEVER;
+	}
+
+	@ConfigItem(
+		position = 0,
+		keyName = "pauseSkillAfter",
+		name = "Auto Pause minutes",
+		description = "Configures how many minutes passes before pausing a skill while in game and there's no XP"
+	)
+	default int pauseSkillAfter()
+	{
+		return 60;
```

Comment:
By default this should never pause

Reply:
Changing to 0, which will act as if disabled.
Will update PR document accordingly.